lib/http-proxy/passes/web-incoming.js

```diff
@@ -132,7 +132,15 @@ web_o = Object.keys(web_o).map(function(pass) {
     req.on('error', proxyError);
 
     // Error Handler
-    proxyReq.on('error', proxyError);
+    proxyReq.on('error', function (err) {
+        if (req.socket.destroyed && err.code === 'ECONNRESET') {
```

Comment:
we should contain this logic in the `proxyError` function and change the error listener for `req` if we are going to keep it